### PR TITLE
design refresh

### LINKED ISSUE #492: Language selector hidden on mobile
image_from_ios

### LINKED ISSUE #529: Rich editor: overflow scroll on code blocks
### Expected behavior

Code blocks should scroll internally, rather than the whole editor field e.g., like github:

```
hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world hello world 
```

### Actual behavior

The whole view is scrollable horizontally:

screenshot 2018-10-10 at 13 12 10

### Steps to reproduce the problem

Insert a long line length code block

### LINKED ISSUE #530: Rich editor: set max-width: 100% on images
### Expected behavior

Images are scaled to the max view size.

### Actual behavior

Images are shown at natural size.

### Steps to reproduce the problem

Insert a large image into the editor.

### PR DESCRIPTION
**Pull request by abovedave**
Lots of UI tweaks to bring Publish more inline with our current aesthetic

Closes #492
Closes #530 
Closes #529

### CODE REVIEW

File: frontend/components/DateTime/DateTime.jsx [hunk 1]
```diff
@@ -102,7 +102,9 @@ export default class DateTime extends Component {
       ? this.getRelativeDate(dateTime.getDate())
       : dateTime.format(format)
 
-    return renderedDate
+    return (
+      <time datetime={dateTime.format(format)} title={dateTime.format(format)}>{renderedDate}</time>
```
Comment (eduardoboucas) [inline frontend/components/DateTime/DateTime.jsx]:
```suggestion
      <time
        datetime={dateTime.format(format)}
        title={dateTime.format(format)}
      >{renderedDate}</time>
```

File: frontend/components/DropdownNative/DropdownNative.css [hunk 2]
```diff
@@ -50,7 +57,13 @@
...
+  padding: 8px 10px;
   position: relative;
   cursor: pointer;
 }
+
+@media (--breakpoint-medium) {
+  .dropdown {
+    padding: 0 30px 0 10px;
+  }
+}
```
Comment (eduardoboucas) [inline frontend/components/DropdownNative/DropdownNative.css]:
```suggestion
}

```

File: frontend/components/FieldNumber/FieldNumberEdit.jsx [hunk 3]
```diff
@@ -105,6 +105,11 @@ export default class FieldNumberEdit extends Component {
     value: null
   }
 
+  constructor(props) {
+    super(props)
```
Comment (eduardoboucas) [inline frontend/components/FieldNumber/FieldNumberEdit.jsx]:
```suggestion
    super(props)

```

File: frontend/components/FieldReference/FieldReference.css [hunk 4]
```diff
@@ -26,21 +28,40 @@
 .value-link {
   display: inline-block;
   border-bottom: 1px solid var(--theme-colour-background-shade-4, #E4E4E4);
-  transition: border-bottom-color 0.1s ease-in-out;
+  /*transition: border-bottom-color 0.1s ease-in-out;*/
   margin: 4px 0;
+  position: relative;
+}
+.value-link:after {
```
Comment (eduardoboucas) [inline frontend/components/FieldReference/FieldReference.css]:
```suggestion

.value-link:after {
```

File: frontend/components/NavItem/NavItem.css [hunk 5]
```diff
@@ -3,46 +3,76 @@
   position: relative;
 }
 
-.container-desktop {
-  display: inline-block;
+@media (--breakpoint-medium) {
+  .container {
+    display: inline-block;
+  }
```
Comment (eduardoboucas) [inline frontend/components/NavItem/NavItem.css]:
```suggestion
  }

```

File: frontend/components/NavItem/NavItem.css [hunk 6]
```diff
@@ -3,46 +3,76 @@
...
+  .nav-item {
+    position: relative;
+    z-index: 3;
+    display: inline-block;
+    font-family: var(--theme-font-family, sans-serif);
+    font-size: 15px;
+    padding: 15px 3px;
+    cursor: pointer;
+    text-decoration: none;
+  }
```
Comment (eduardoboucas) [inline frontend/components/NavItem/NavItem.css]:
```suggestion
  }

```

File: frontend/components/NavItem/NavItem.css [hunk 7]
```diff
@@ -3,46 +3,76 @@
...
+    display: block;
+  }
+
+@media (--breakpoint-small) {
+  .nav-item {
+    font-weight: bold;
+    color: #687077;
+    padding: 15px 0 8px 0;
+    display: block;
+  }
```
Comment (eduardoboucas) [inline frontend/components/NavItem/NavItem.css]:
```suggestion
  }

```

File: frontend/components/NavItem/NavItem.css [hunk 8]
```diff
@@ -3,46 +3,76 @@
...
+  .nav-item span {
+    padding: 0;
+  }
```
Comment (eduardoboucas) [inline frontend/components/NavItem/NavItem.css]:
```suggestion
  }

```

File: frontend/components/NavItem/NavItem.css [hunk 9]
```diff
@@ -3,46 +3,76 @@
...
+  .nav-item-active span {
+    color: #ffbf0e;
+    background: none;
+  }
```
Comment (eduardoboucas) [inline frontend/components/NavItem/NavItem.css]:
```suggestion
  }

```

File: frontend/components/RichEditor/RichEditor.css [hunk 10]
```diff
@@ -147,10 +148,23 @@
...
-  background-color: var(--theme-colour-background-shade-1, #fafafa);
-  position: relative;
+  padding: 7px 4px 4px;
+  border-bottom: 1px solid #eee;
   z-index: 1;
+  position: sticky;
+  top: 0;
+  background: #fff;
+  color: var(--theme-colour-primary, #000000);
+}
```
Comment (eduardoboucas) [inline frontend/components/RichEditor/RichEditor.css]:
```suggestion
}

```

File: frontend/containers/DocumentListToolbar/DocumentListToolbar.jsx [hunk 11]
```diff
@@ -102,29 +102,17 @@ class DocumentListToolbar extends Component {
...
+            <span class={styles['count-label']}>
+              <span>Showing </span>
+              <strong>{`${metadata.offset + 1}-${Math.min(metadata.offset + metadata.limit, metadata.totalCount)} `}</strong>
+              of <strong>{metadata.totalCount}</strong>
+            </span>
           </div>
-        </div>
-
+        )}
+
```
Comment (eduardoboucas) [inline frontend/containers/DocumentListToolbar/DocumentListToolbar.jsx]:
```suggestion

```

File: frontend/containers/Header/Header.css [hunk 12]
```diff
@@ -65,35 +64,60 @@
...
 
-.content-compact {
-  position: fixed;
-  width: 100%;
-  transition: transform 0.3s;
-  transform: translateX(-100%);
+.controls {
+  padding: 10px;
+}
+@media (--breakpoint-medium) {
```
Comment (eduardoboucas) [inline frontend/containers/Header/Header.css]:
```suggestion

@media (--breakpoint-medium) {
```

File: frontend/containers/Header/Header.css [hunk 13]
```diff
@@ -65,35 +64,60 @@
...
-.inner-content-compact {
-  max-height: 400px;
-  overflow: auto;
+.toggle-icon {
+  float: right;
+  padding: 12px 12px 12px 12px;
+  cursor: pointer;
+  display: block;
+}
+.toggle-icon:hover {
```
Comment (eduardoboucas) [inline frontend/containers/Header/Header.css]:
```suggestion

.toggle-icon:hover {
```

File: frontend/containers/Header/Header.css [hunk 14]
```diff
@@ -65,35 +64,60 @@
...
+.icon-open,
+.icon-close {
+  background-position: 50%;
+  background-image: url('/public/images/icon-burger.svg');
+  background-size: auto 100%;
+  font-size: 0;
+  width: 21px; height: 21px;
+  display: block;
+}
+.icon-close {
```
Comment (eduardoboucas) [inline frontend/containers/Header/Header.css]:
```suggestion

.icon-close {
```

File: frontend/containers/SignIn/SignIn.css [hunk 15]
```diff
@@ -38,20 +38,22 @@
...
 
-@media (--breakpoint-medium) {
-  .input {
-    width: 50%;
-  }
+.input label {
+  border: 0;
+  border-radius: 0 0 3px 3px;
+}
+.input:first-of-type label {
```
Comment (eduardoboucas) [inline frontend/containers/SignIn/SignIn.css]:
```suggestion

.input:first-of-type label {
```

File: frontend/components/Button/Button.jsx [hunk 16]
```diff
@@ -121,7 +121,7 @@ export default class Button extends Component {
 
     if (type === 'mock') {
       return (
-        <span class={buttonStyle.getClasses()}>{children}</span>
+        <a href="#" class={buttonStyle.getClasses()}>{children}</a>
```
[the remark this answers is not in the document]
Comment (abovedave) [reply frontend/components/Button/Button.jsx]: When in groups I wanted to be able to target with a `:first-of-type`, regardless as to if it's a reall `a` button or a `span` (as it was previously). Open to suggestions!

**Pull request merged**